src/text/style.ts

```diff
@@ -0,0 +1,52 @@
+import css from 'styled-jsx/css'
+import { color, font, fontWeight } from '_utils/branding'
+
+export default css`
+  .kirk-text {
+    margin: 0;
+    font-weight: ${fontWeight.regular};
+  }
+
+  .kirk-text.kirk-text-button {
+    color: ${color.primaryText};
+    font-size: ${font.base.size};
+    line-height: ${font.base.lineHeight};
+  }
+
+  .kirk-text-body,
+  .kirk-text-bodyStrong {
+    color: ${color.secondaryText};
+    font-size: ${font.base.size};
+    line-height: ${font.base.lineHeight};
+  }
+
+  .kirk-text-caption {
+    color: ${color.secondaryText};
+    font-size: ${font.s.size};
+    line-height: ${font.s.lineHeight};
+  }
+
+  .kirk-text-display1 {
```

Comment:
If this display class is related to the size it should be named like .kirk-text-xxl and .kirk-text-xl